src/pick.c

```diff
@@ -65,19 +65,21 @@ static const struct choice	*selected_choice(void);
 static void			 filter_choices(void);
 static void			 score(struct choice *);
 static int			 min_match(const char *, size_t, size_t *, size_t *);
-static char			*strcasechr(const char *, char);
+static char			 *strcasechr(const char *, const char *);
```

Comment:
On Fri, Apr 08, 2016 at 01:24:34AM -0700, Calle Erlandsson wrote:
> > @@ -65,19 +65,21 @@ static const struct choice	*selected_choice(void);
> >  static void			 filter_choices(void);
> >  static void			 score(struct choice *);
> >  static int			 min_match(const char *, size_t, size_t *, size_t *);
> > -static char			*strcasechr(const char *, char);
> > +static char			 *strcasechr(const char *, const char *);
> 
> Should we remove the space in front of the first `*` on this line?
> 
> In the past, I've tried to align the declared function names for readability. The added space in front of the functions not returning a pointer is to account for the asterisk for those that do. This is described in OpenBSD's style(9):
> 
> > Prototypes may have an extra space after a tab to enable function names to line up:
> > ```
> > static char	*function(int, const char *); 
> > static void	 usage(void);
> > ```

Nice catch, fixed in 0bd3350.